vivisect/parsers/blob.py

```diff
@@ -31,6 +31,7 @@ def parseFd(vw, fd, filename=None, baseaddr=None):
     vw.setMeta('DefaultCall', archcalls.get(arch,'unknown'))
 
     bytez =  fd.read() 
+    vw.setFileMeta('sha256', v_parsers.sha256Bytes(bytez))
```

Comment:
Also need the filename here as the first parameter (that's what the tests are failing on)

Reply:
yup, <facepalm>